projects/demo/project.json

```diff
@@ -14,7 +14,7 @@
             }
         },
         "build": {
-            "executor": "@angular-builders/custom-webpack:browser",
+            "executor": "@nx/angular:webpack-browser",
```

Comment:
Should we also change executor for `server`-target ?

https://nx.dev/nx-api/angular/executors/webpack-server

Reply:
there are some problems with nx executor, let's do it in a separate pr